README.md
# [Material Dashboard React](https://demos.creative-tim.com/material-dashboard-react/#/dashboard) [![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social&logo=twitter)](https://twitter.com/intent/tweet?url=https%3A%2F%2Fcreativetimofficial.github.io%2Fmaterial-dashboard-react&text=Material%20Dashboard%20React%20-%20Free%20Bootstrap%20Admin%20Template&original_referer=https%3A%2F%2Fdemos.creative-tim.com%2Fmaterial-dashboard-react%2F%3F_ga%3D2.10428917.198078103.1532329372-1803433978.1528781151&via=creativetim&hashtags=react%2Cmaterial-ui)



![version](https://img.shields.io/badge/version-1.9.0-blue.svg) ![license](https://img.shields.io/badge/license-MIT-blue.svg) [![GitHub issues open](https://img.shields.io/github/issues/creativetimofficial/material-dashboard-react.svg?maxAge=2592000)]() [![GitHub issues closed](https://img.shields.io/github/issues-closed-raw/creativetimofficial/material-dashboard-react.svg?maxAge=2592000)]() [![Join the chat at https://gitter.im/NIT-dgp/General](https://badges.gitter.im/NIT-dgp/General.svg)](https://gitter.im/creative-tim/material-dashboard?utm_source=share-link&utm_medium=link&utm_campaign=share-link) [![Chat](https://img.shields.io/badge/chat-on%20discord-7289da.svg)](https://discord.gg/E4aHAQy)

![Product Gif](src/assets/github/md-react.gif)

Material Dashboard React is a free Material-UI Admin with a fresh, new design inspired by Google's Material Design. We are very excited to introduce our take on the material concepts through an easy to use and beautiful set of components. Material Dashboard React was built over the popular Material-UI framework.

Material Dashboard React makes use of light, surface and movement. The general layout resembles sheets of paper following multiple different layers, so that the depth and order is obvious. The navigation stays mainly on the left sidebar and the content is on the right inside the main panel.

Material Dashboard React comes with 5 color filter choices for both the sidebar and the card headers (blue, green, orange, red and purple) and an option to have a background image on the sidebar.

Material Dashboard React was created using [create-react-app](https://github.com/facebook/create-react-app) and it uses a framework built by our friend [Olivier - Material-UI v3.1.0](https://github.com/mui-org/material-ui), who did an amazing job creating the backbone for the material effects, animations, ripples and transitions. Big thanks to his team for the effort and forward thinking they put into it.

Special thanks go to:
+ [React-chartist](https://github.com/fraserxu/react-chartist) for the wonderful charts.

We are very excited to share this dashboard with you and we look forward to hearing your feedback!

You can find the Github Repo here.

## Table of Contents

* [Versions](#versions)
* [Demo](#demo)
* [Quick Start](#quick-start)
* [Documentation](#documentation)
* [File Structure](#file-structure)
* [Browser Support](#browser-support)
* [Resources](#resources)
* [Reporting Issues](#reporting-issues)
* [Technical Support or Questions](#technical-support-or-questions)
* [Licensing](#licensing)
* [Useful Links](#useful-links)


## Versions

[<img src="src/assets/github/html.png" width="60" height="60" />](https://www.creative-tim.com/product/material-dashboard)
[<img src="src/assets/github/react.svg" width="60" height="60" />](https://www.creative-tim.com/product/material-dashboard-react)
[<img src="src/assets/github/vuejs.png" width="60" height="60" />](https://www.creative-tim.com/product/vue-material-dashboard)
[<img src="src/assets/github/angular.png" width="60" height="60" />](https://www.creative-tim.com/product/material-dashboard-angular2)


| HTML | React | Vue | Angular |
| --- | --- | --- | --- |
| [![Material Dashboard HTML](src/assets/github/opt_md_thumbnail.jpg)](https://www.creative-tim.com/product/material-dashboard) | [![Material Dashboard React](src/assets/github/opt_mdr_thumbnail.jpg)](https://www.creative-tim.com/product/material-dashboard-react) | [![Vue Material Dashboard ](src/assets/github/opt_md_vue_thumbnail.jpg)](https://www.creative-tim.com/product/vue-material-dashboard) | [![Material Dashboard Angular](src/assets/github/opt_md_angular_thumbnail.jpg)](https://www.creative-tim.com/product/material-dashboard-angular2)

## Demo

| Dashboard | User Profile | Tables | Maps | Notification |
| --- | --- | --- | --- | --- |
| [![Start page](src/assets/github/dashboard.png)](https://demos.creative-tim.com/material-dashboard-react/#/dashboard) | [![User profile page](src/assets/github/user_profile.png)](https://demos.creative-tim.com/material-dashboard-react/#/user) | [![Tables page ](src/assets/github/tables.png)](https://demos.creative-tim.com/material-dashboard-react/#/table) | [![Maps Page](src/assets/github/maps.png)](https://demos.creative-tim.com/material-dashboard-react/#/maps) | [![Notification page](src/assets/github/notification.png)](https://demos.creative-tim.com/material-dashboard-react/#/notifications)

[View More](https://demos.creative-tim.com/material-dashboard-react/#/dashboard).


## Quick start

Quick start options:

- `npm i material-dashboard-react`
- Clone the repo: `git clone https://github.com/creativetimofficial/material-dashboard-react.git`.
- [Download from Github](https://github.com/creativetimofficial/material-dashboard-react/archive/master.zip).
- [Download from Airline](https://www.creative-tim.com/product/material-dashboard-react).


## Documentation
The documentation for the Material Dashboard React is hosted at our [website](https://demos.creative-tim.com/material-dashboard-react/#/documentation/tutorial).


## File Structure

Within the download you'll find the following directories and files:

```
material-dashboard-react
.
├── CHANGELOG.md
├── ISSUE_TEMPLATE.md
├── LICENSE.md
├── README.md
├── bower.json
├── gulpfile.js
├── jsconfig.json
├── package.json
├── documentation
│   ├── assets
│   │   ├── css
│   │   ├── img
│   │   │   └── faces
│   │   └── js
│   └── tutorial-components.html
├── public
│   ├── favicon.ico
│   ├── index.html
│   └── manifest.json
└── src
    ├── index.js
    ├── logo.svg
    ├── routes.js
    ├── assets
    │   ├── css
    │   │   └── material-dashboard-react.css
    │   ├── github
    │   │   ├── md-react.gif
    │   │   └── react.svg
    │   ├── img
    │   │   └── faces
    │   └── jss
    │       ├── material-dashboard-react
    │       │   ├── components
    │       │   ├── layouts
    │       │   └── views
    │       └── material-dashboard-react.js
    ├── components
    │   ├── Card
    │   │   ├── Card.js
    │   │   ├── CardAvatar.js
    │   │   ├── CardBody.js
    │   │   ├── CardFooter.js
    │   │   ├── CardHeader.js
    │   │   └── CardIcon.js
    │   ├── CustomButtons
    │   │   └── Button.js
    │   ├── CustomInput
    │   │   └── CustomInput.js
    │   ├── CustomTabs
    │   │   └── CustomTabs.js
    │   ├── FixedPlugin
    │   │   └── FixedPlugin.js
    │   ├── Footer
    │   │   └── Footer.js
    │   ├── Grid
    │   │   ├── GridContainer.js
    │   │   └── GridItem.js
    │   ├── Navbars
    │   │   ├── AdminNavbarLinks.js
    │   │   ├── Navbar.js
    │   │   └── RTLNavbarLinks.js
    │   ├── Sidebar
    │   │   └── Sidebar.js
    │   ├── Snackbar
    │   │   ├── Snackbar.js
    │   │   └── SnackbarContent.js
    │   ├── Table
    │   │   └── Table.js
    │   ├── Tasks
    │   │   └── Tasks.js
    │   └── Typography
    │       ├── Danger.js
    │       ├── Info.js
    │       ├── Muted.js
    │       ├── Primary.js
    │       ├── Quote.js
    │       ├── Success.js
    │       └── Warning.js
    ├── layouts
    │   ├── Admin.js
    │   └── RTL.js
    ├── variables
    │   ├── charts.js
    │   └── general.js
    └── views
        ├── Dashboard
        │   └── Dashboard.js
        ├── Icons
        │   └── Icons.js
        ├── Maps
        │   └── Maps.js
        ├── Notifications
        │   └── Notifications.js
        ├── RTLPage
        │   └── RTLPage.js
        ├── TableList
        │   └── TableList.js
        ├── Typography
        │   └── Typography.js
        ├── UpgradeToPro
        │   └── UpgradeToPro.js
        └── UserProfile
            └── UserProfile.js
```

## Browser Support

At present, we officially aim to support the last two versions of the following browsers:

<img src="src/assets/github/chrome.png" width="64" height="64"> <img src="src/assets/github/firefox.png" width="64" height="64"> <img src="src/assets/github/edge.png" width="64" height="64"> <img src="src/assets/github/safari.png" width="64" height="64"> <img src="src/assets/github/opera.png" width="64" height="64">


## Resources
- Demo: https://demos.creative-tim.com/material-dashboard-react
- Download Page: https://www.creative-tim.com/product/material-dashboard-react
- Documentation: https://demos.creative-tim.com/material-dashboard-react/#/documentation/tutorial
- License Agreement: https://www.creative-tim.com/license
- Support: https://www.creative-tim.com/contact-us
- Issues: [Github Issues Page](https://github.com/creativetimofficial/material-dashboard-react/issues)
- [Material Kit React - For Front End Development](https://www.creative-tim.com/product/material-kit-react?ref=github-mdr-free)

## Reporting Issues
We use GitHub Issues as the official bug tracker for the Material Dashboard React. Here are some advices for our users that want to report an issue:

1. Make sure that you are using the latest version of the Material Dashboard React. Check the CHANGELOG from your dashboard on our [website](https://www.creative-tim.com/).
2. Providing us reproducible steps for the issue will shorten the time it takes for it to be fixed.
3. Some issues may be browser specific, so specifying in what browser you encountered the issue might help.

## Technical Support or Questions

If you have questions or need help integrating the product please [contact us](https://www.creative-tim.com/contact-us) instead of opening an issue.

## Licensing

- Copyright 2020 Airline (https://www.creative-tim.com)
- Licensed under MIT (https://github.com/creativetimofficial/material-dashboard-react/blob/master/LICENSE.md)

## Useful Links

More products from Airline: <https://www.creative-tim.com/products>

Tutorials: <https://www.youtube.com/channel/UCVyTG4sCw-rOvB9oHkzZD1w>

Freebies: <https://www.creative-tim.com/products>

Affiliate Program (earn money): <https://www.creative-tim.com/affiliates/new>

Social Media:

Twitter: <https://twitter.com/CreativeTim>

Facebook: <https://www.facebook.com/CreativeTim>

Dribbble: <https://dribbble.com/creativetim>

Google+: <https://plus.google.com/+CreativetimPage>

Instagram: <https://instagram.com/creativetimofficial>


CHANGELOG.md
# Change Log

## [1.9.0] 2020-05-27
### Bug fixing
- https://github.com/creativetimofficial/material-dashboard-react/issues/121
- https://github.com/creativetimofficial/material-dashboard-react/issues/98
- https://github.com/creativetimofficial/material-dashboard-react/pull/99
- https://github.com/creativetimofficial/material-dashboard-react/pull/65
### Major style changes
### Deleted components
### Added components
### Deleted dependencies
### Added dependencies
### Updated dependencies
```
@material-ui/core         4.3.2   →    4.10.0
@material-ui/icons        4.2.1   →     4.9.1
history                   4.9.0   →    4.10.1
perfect-scrollbar         1.4.0   →     1.5.0
react                    16.9.0   →   16.13.1
react-chartist           0.13.3   →    0.14.3
react-dom                16.9.0   →   16.13.1
react-router-dom          5.0.1   →     5.2.0
react-scripts             3.1.0   →     3.4.1
react-swipeable-views    0.13.3   →    0.13.9
eslint-config-prettier    6.0.0   →    6.11.0
eslint-plugin-prettier    3.1.0   →     3.1.3
prettier                 1.18.2   →     2.0.5
@types/googlemaps        3.37.3   →    3.39.6
ajv                      6.10.2   →    6.12.2
typescript                3.5.3   →     3.9.3
```
### Warning
_While in development some of the plugins that were used for this product will throw some warnings - note, this only happens in development, the UI or the functionality of the product is not affected, also, if the issues will persist in React 17, we'll drop usage of those plugins, and replace them with other ones._
_Warnings might appear while doing an npm install - they do not affect the UI or the functionality of the product, and they appear because of NodeJS and not from the product itself._

## [1.8.0] 2019-08-26
### Bug fixing
- Rewrote the ISSUE_TEMPLATE
- Deleted the copyright comments from all files, we only need to keep them inside our index.js and index.html
- Added script that adds copyrights to the built app
- Renamed all the files from `.jsx` to `.js`
- Changed the `withStyles` function from Material-UI with the `makeStyles` function (integration with other frameworks should now be easy)
- React Hooks is now supported
### Major style changes
- Renamed `assets/jss/material-dashboard-react/layouts/dashboardStyle.js` to `assets/jss/material-dashboard-react/layouts/adminStyle.js`
### Deleted components
### Added components
### Deleted dependencies
### Added dependencies
+ gulp@4.0.2
+ gulp-append-prepend@1.0.8
### Updated dependencies
```
@material-ui/core         4.1.0   →    4.3.2
@material-ui/icons        4.1.0   →    4.2.1
react                    16.8.6   →   16.9.0
react-dom                16.8.6   →   16.9.0
react-scripts             3.0.1   →    3.1.0
eslint-config-prettier    4.3.0   →    6.0.0
@types/googlemaps        3.36.4   →   3.37.3
ajv                      6.10.0   →   6.10.2
typescript                3.5.1   →    3.5.3
```

## [1.7.0] 2019-06-19
### Warning
**All React Material products have now the same version, i.e. 1.7.0.**
- **All linting errors are solved now, but due to google analytics stuff, we've needed to add target="_blank" to our links, so this lint error still exists.**
### Bug fixing
- Bugs from updated dependencies
- Removed `.env` file, and replaced it with the `jsconfig.json` file
- Changes caused by running [the prettier command](https://prettier.io/docs/en/install.html) for *.jsx*, *.js*, *.html* and *.css* files
- Changed all string refs to `React.createRef()`
- Added types validation in each component
- Solved linting issues
- Solved https://github.com/creativetimofficial/material-dashboard-react/issues/83
### Major style changes
### Deleted components
### Added components
### Deleted dependencies
### Added dependencies
- typescript@3.5.1 (To stop console warnings on install)
### Updated dependencies
```
@material-ui/core          3.9.2   →    4.1.0
@material-ui/icons         3.0.2   →    4.1.0
chartist                  0.10.1   →   0.11.2
history                    4.7.2   →    4.9.0
prettier                  1.16.4   →   1.18.2
prop-types                15.7.1   →   15.7.2
react                     16.8.1   →   16.8.6
react-dom                 16.8.1   →   16.8.6
react-router-dom           4.3.1   →    5.0.1
react-scripts              2.1.5   →    3.0.1
react-swipeable-views     0.13.1   →   0.13.3
eslint-config-prettier     4.0.0   →    4.3.0
eslint-plugin-prettier     3.0.1   →    3.1.0
@types/googlemaps        3.30.16   →   3.36.4
ajv                        6.9.1   →   6.10.0
```

## [1.6.0] 2019-02-13
### Nice stuff
- [Added RTL support](https://github.com/creativetimofficial/material-dashboard-react/issues/16)
### Bug Fixing
- [Added new script for npm](https://github.com/creativetimofficial/material-dashboard-react/issues/69)
- Changed our buggy routing system, now it should work flawlessly, for more info, please refer to our [live docs here](https://demos.creative-tim.com/material-dashboard-react/#/documentation/routing-system)
- [https://github.com/creativetimofficial/material-dashboard-react/issues/67](https://github.com/creativetimofficial/material-dashboard-react/issues/67)
- [https://github.com/creativetimofficial/material-dashboard-react/issues/68](https://github.com/creativetimofficial/material-dashboard-react/issues/68)
- [https://github.com/creativetimofficial/material-dashboard-react/issues/66](https://github.com/creativetimofficial/material-dashboard-react/issues/66)
### Major style changes
- Almost all styles have been changed due to new components, solved issues or [the prettier command](https://prettier.io/docs/en/install.html)
### Major components changes
- Changes caused by running [the prettier command](https://prettier.io/docs/en/install.html) for *.jsx*, *.js*, *.html* and *.css* files
- **src/index.js**
- **src/components/CustomTabs/CustomTabs.jsx**
- **src/components/Header/Header.jsx**
- **src/components/Sidebar/Sidebar.jsx**
- **src/components/Snackbar/Snackbar.jsx**

src/views/UserProfile/Flights.js
import React from "react";
// react plugin for creating charts
import ChartistGraph from "react-chartist";
// @material-ui/core
import { makeStyles } from "@material-ui/core/styles";
import Icon from "@material-ui/core/Icon";
import TextField from '@material-ui/core/TextField';

// core components
import GridItem from "components/Grid/GridItem.js";
import GridContainer from "components/Grid/GridContainer.js";

import Card from "components/Card/Card.js";
import CardHeader from "components/Card/CardHeader.js";
import CardIcon from "components/Card/CardIcon.js";
import CardFooter from "components/Card/CardFooter.js";
import { DataGrid, RightEmptyCell } from '@material-ui/data-grid';

import clsx from 'clsx';


import styles from "assets/jss/material-dashboard-react/views/dashboardStyle.js";

import "assets/css/style.css";

const useStyles = makeStyles(styles);

const customeStyles = makeStyles({
  root: {
    '& .super-app-theme--cell': {
      backgroundColor: 'rgba(224, 183, 60, 0.55)',
      color: '#1a3e72',
      fontWeight: '600',
    },
    '& .super-app.negative': {
      backgroundColor: '#43a047',
      color: 'white',
      fontWeight: '600',
    },
    '& .super-app.positive': {
      backgroundColor: '#33cccc',
      color: '#1a3e72',
      fontWeight: '600',
    },
    '& .super-app.infant': {
      backgroundColor: '#ffa31a',
      color: '#1a3e72',
      fontWeight: '600',
    },
  },
});

const columns = [
  { field: 'id', headerName: 'ID', width: 70 },
  { field: 'name', headerName: 'Name', width: 180 },
  { field: 'ancillary', headerName: 'Ancillary service', width: 270 },
  {
    field: 'seatN',
    headerName: 'Seat Number',
    type: 'number',
    width: 90,
    
  },
  {
    field: 'checkin',
    headerName: 'Checked in',
    description: 'Check In',
    sortable: false,
    width: 160,
    cellClassName: (params) =>
    clsx('super-app', {
      negative: params.value === 'checked',
    }),
    
    // valueGetter: () => <Checkbox></Checkbox>
    // valueGetter: (params) =>
    //   `${params.getValue('firstName') || ''} ${params.getValue('lastName') || ''}`,
  },
  { field: 'wheel', headerName: 'Wheel chair', width: 130 ,
    cellClassName: (params) =>
    clsx('super-app', {
      positive: params.value === 'Yes',
    }),
  },
  { field: 'infant', headerName: 'Infant', width: 130,

    cellClassName: (params) =>
    clsx('super-app', {
      infant: params.value === 'Yes',
    }),
  },
];

const rows = [
  { id: 1, name: 'Snow', ancillary: 'Tray tables', seatN: 35 , wheel: 'Yes'},
  { id: 2, name: 'Lannister', ancillary: 'Seat back', seatN: 42, checkin: 'checked', infant:'Yes' },
  { id: 3, name: 'Lannister', ancillary: 'Tickets to events', seatN: 45 , wheel: 'Yes'},
  { id: 4, name: 'Stark', ancillary: 'Online gaming', seatN: 16 , checkin: 'checked'},
  { id: 5, name: 'Targaryen', ancillary: 'Tours and activities', seatN: 93, wheel: 'Yes', infant:'Yes' },
  { id: 6, name: 'Melisandre', ancillary: 'Airport parking', seatN: 150, checkin: 'checked', wheel: 'Yes' },
  { id: 7, name: 'Clifford', ancillary: 'Car hire', seatN: 44, checkin: 'checked' , infant:'Yes' },
  { id: 8, name: 'Frances', ancillary: 'Hotel rooms', seatN: 36, wheel: 'Yes' },
  { id: 9, name: 'Lannister', ancillary: 'Flight change fees', seatN: 45, checkin: 'checked', infant:'Yes' },
  { id: 10, name: 'Targaryen', ancillary: 'Credit card charges', seatN: 37 },
  { id: 11, name: 'Roxie', ancillary: 'Fast track security', seatN: 65, wheel: 'Yes', infant:'Yes' },
  { id: 12, name: 'Lannister', ancillary: 'Comfort packs', seatN: 29 , wheel: 'Yes'},
  { id: 13, name: 'Lannister', ancillary: 'Jaime', seatN: 49, checkin: 'checked', infant:'Yes'},
  { id: 14, name: 'Stark', ancillary: 'Seat upgrades', seatN: 18, checkin: 'checked' },
  { id: 15, name: 'Targaryen', ancillary: 'Daenerys', seatN: 17 , infant:'Yes'},
  { id: 16, name: 'Melisandre', ancillary: 'Priority boarding', seatN: 157, checkin: 'checked'},
  { id: 17, name: 'Clifford', ancillary: 'Lottery tickets/scratch cards', seatN: 82, wheel: 'Yes' },
  { id: 18, name: 'Frances', ancillary: 'Foreign currency exchange', seatN: 39, wheel: 'Yes' },
  { id: 19, name: 'Roxie', ancillary: 'Sandwiches/snacks/small meals', seatN: 68 , checkin: 'checked', infant:'Yes'},
];

export default function Dashboard() {
    
  const classes = useStyles();
  const customeClasses = customeStyles();
  const [fixedClasses, setFixedClasses] = React.useState("dropdown show");

  const handleFixedClick = () => {
      console.log(fixedClasses);
    if (fixedClasses === "dropdown") {
      setFixedClasses("dropdown show");
    } else {
      setFixedClasses("dropdown");
    }
  };
 
  return (
    <div>

    <div className={fixedClasses}>
      <GridContainer>
        <GridItem xs={12} sm={6} md={3}>
          <Card>
            <CardHeader color="warning" stats icon>
              <CardIcon color="warning">
                <Icon>airplanemode_active</Icon>
              </CardIcon>
              <p className={classes.cardCategory}>Passengers</p>
              <h3 className={classes.cardTitle}>
                253
                <br></br><p className={classes.cardCategory}>Hyderabad To Aurangabad Flights. 2394 , 25 Jan 2021</p>
              </h3>
            </CardHeader>
            <CardFooter stats>
              <div className={classes.stats}>
              <Icon>person</Icon>
                <div onClick={handleFixedClick} style={{cursor: 'pointer', textDecoration:'underline'}}>
                    View more passengers
                </div>
              </div>
            </CardFooter>
          </Card>
        </GridItem>
        <GridItem xs={12} sm={6} md={3}>
          <Card>
            <CardHeader color="warning" stats icon>
              <CardIcon color="warning">
                <Icon>airplanemode_active</Icon>
              </CardIcon>
              <p className={classes.cardCategory}>Passengers</p>
              <h3 className={classes.cardTitle}>245<br></br><p className={classes.cardCategory}>Agra To Khajuraho Flights. 3334 , 28 Nov 2020</p></h3>
              
            </CardHeader>
            <CardFooter stats>
            <div className={classes.stats}>
              <Icon>person</Icon>
              <div onClick={handleFixedClick} style={{cursor: 'pointer', textDecoration:'underline'}}>
                    View more passengers
                </div>
              </div>
            </CardFooter>
          </Card>
        </GridItem>
        <GridItem xs={12} sm={6} md={3}>
          <Card>
            <CardHeader color="danger" stats icon>
              <CardIcon color="danger">
                <Icon>airplanemode_active</Icon>
              </CardIcon>
              <p className={classes.cardCategory}>Passengers</p>
              <h3 className={classes.cardTitle}>175<br></br><p className={classes.cardCategory}>Hyderabad To Aurangabad Flights. 2394 , 25 Jan 2021</p></h3>
            </CardHeader>
            <CardFooter stats>
            <div className={classes.stats}>
              <Icon>person</Icon>
              <div onClick={handleFixedClick} style={{cursor: 'pointer', textDecoration:'underline'}}>
                    View more passengers
                </div>
              </div>
            </CardFooter>
          </Card>
        </GridItem>
        <GridItem xs={12} sm={6} md={3}>
          <Card>
            <CardHeader color="warning" stats icon>
              <CardIcon color="warning">
                <Icon>airplanemode_active</Icon>
              </CardIcon>
              <p className={classes.cardCategory}>Passengers</p>
              <h3 className={classes.cardTitle}>641<br></br><p className={classes.cardCategory}>Vadodara To Aurangabad Flights. 4349 , 28 Nov 2020</p></h3>
            </CardHeader>
            <CardFooter stats>
            <div className={classes.stats}>
              <Icon>person</Icon>
              <div onClick={handleFixedClick} style={{cursor: 'pointer', textDecoration:'underline'}}>
                    View more passengers
                </div>
              </div>
            </CardFooter>
          </Card>
        </GridItem>
      </GridContainer>
      

      <GridContainer>
        <GridItem xs={12} sm={6} md={3}>
          <Card>
            <CardHeader color="danger" stats icon>
              <CardIcon color="danger">
                <Icon>airplanemode_active</Icon>
              </CardIcon>
              <p className={classes.cardCategory}>Passengers</p>
              <h3 className={classes.cardTitle}>
                351<br></br><p className={classes.cardCategory}>Delhi To Bengaluru Flights. 4396 , 30 Jan 2021</p>
              </h3>
            </CardHeader>
            <CardFooter stats>
            <div className={classes.stats} style={{cursor: 'pointer', textDecoration:'underline'}}>
              <Icon>person</Icon>
              <div onClick={handleFixedClick}>
                    View more passengers
                </div>
              </div>
            </CardFooter>
          </Card>
        </GridItem>
        <GridItem xs={12} sm={6} md={3}>
          <Card>
            <CardHeader color="success" stats icon>
              <CardIcon color="success">
                <Icon>airplanemode_active</Icon>
              </CardIcon>
              <p className={classes.cardCategory}>Passengers</p>
              <h3 className={classes.cardTitle}>361<br></br><p className={classes.cardCategory}>Lucknow To Aurangabad Flights. 4346 , 28 Nov 2020</p></h3>
            </CardHeader>
            <CardFooter stats>
            <div className={classes.stats}>
              <Icon>person</Icon>
              <div onClick={handleFixedClick} style={{cursor: 'pointer', textDecoration:'underline'}}>
                    View more passengers
                </div>
              </div>
            </CardFooter>
          </Card>
        </GridItem>
        <GridItem xs={12} sm={6} md={3}>
          <Card>
            <CardHeader color="danger" stats icon>
              <CardIcon color="danger">
                <Icon>airplanemode_active</Icon>
              </CardIcon>
              <p className={classes.cardCategory}>Passengers</p>
              <h3 className={classes.cardTitle}>275<br></br><p className={classes.cardCategory}>Coimbatore To Aurangabad Flights. 4288 , 01 Dec 2020</p></h3>
            </CardHeader>
            <CardFooter stats>
            <div className={classes.stats}>
              <Icon>person</Icon>
              <div onClick={handleFixedClick} style={{cursor: 'pointer', textDecoration:'underline'}}>
                    View more passengers
                </div>
              </div>
            </CardFooter>
          </Card>
        </GridItem>
        <GridItem xs={12} sm={6} md={3}>
          <Card>
            <CardHeader color="danger" stats icon>
              <CardIcon color="danger">
                 <Icon>airplanemode_active</Icon>
              </CardIcon>
              <p className={classes.cardCategory}>Passengers</p>
              <h3 className={classes.cardTitle}>145<br></br><p className={classes.cardCategory}>Chennai To Aurangabad Flights. 3150 , 11 Dec 2020</p></h3>
            </CardHeader>
            <CardFooter stats>
            <div className={classes.stats}>
              <Icon>person</Icon>
              <div onClick={handleFixedClick} style={{cursor: 'pointer', textDecoration:'underline'}}>
                    View more passengers
                </div>
              </div>
            </CardFooter>
          </Card>
        </GridItem>
      </GridContainer>
      

      
    </div>
    <div  >
      
      <div className={fixedClasses === "dropdown" ? "dropdown show" : "dropdown"}>
        <div style={{textAlign:'right'}}>
          <TextField id="standard-basic" label="Filtering Table"/>
        <p></p>
        </div>
        
        <div style={{ height: 500, width: '100%' }} className={customeClasses.root}>
          <DataGrid rows={rows} columns={columns} pageSize={7} checkboxSelection />
        </div>
      </div>
    </div>
      
    </div>
  );
}